dao.py

```diff
@@ -619,6 +618,25 @@ def get_user_by_session_id_or_none(self, session_id):
     def get_user_by_region(self, regions):
         pass
 
+    def get_is_superadmin(self, user_id):
+        user = None
+        if self.users_col.find_one({'_id': user_id}):
+            user = M.User.load(self.users_col.find_one({'_id': user_id}))
+            return user.admin_level == 'SUPER'
+        else:
+            return False
+
+    def set_user_admin_level(self, user_id, admin_level):
+        if type(admin_level) is not M.AdminLevels:
+            raise Exception('Submitted admin level is not of correct type')
+        if self.users_col.find_one({'_id': user_id}):
```

Comment:
right now this way of updating bypasses the ORM completely and won't trigger any of the ORMs validators. the correct way to do this sort of thing is 1. read the user data into a User object (as is done in get_user_by_id_or_none), 2. update whatever properties of the User object, 3. then write the User object back into the database using `self.users_col.update({'_id': user.id}, user.dump(context='db'))`.